conf/application.conf

```diff
@@ -173,4 +173,5 @@ feature-switch {
     stubDes = false
     Api1365Version = "R11"
     Api1363Version = "R10"
+    enableAnnualAccounting = true
```

Comment:
After running the application it looks like this needs to be disabled as the frontend doesn't support the new return periods yet. This will need updating in app-config-base as well